mywebserver/notes/tests.py

```diff
@@ -1,3 +1,24 @@
-from django.test import TestCase
+# mywebserver/notes/tests.py
+from django.urls import reverse
+from django.test import Client
+import pytest
 
-# Create your tests here.
+client = Client()
+
+
+@pytest.mark.django_db
```

Comment:
To check routing it would be enough to use `@pytest.mark.urls` since you are not working with the database, if you need access to the database you can use both.

Please review our work in class one more time.